tfjs-backend-webgpu/src/kernels/binary_op_shared_webgpu.ts

```diff
@@ -57,10 +60,20 @@ export class BinaryOpSharedProgram implements WebGPUProgram {
     this.dispatch = computeDispatch(
```

Comment:
Can we do the reshape dispatch in `computeDispatch` rather than do it in a separate function?